keyboards/viendi8l/config.h

```diff
@@ -0,0 +1,72 @@
+/*
+ Copyright 2022 Gondolindrim <gondolindrim@acheronproject.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+#include "common_config.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x8673 // VI for Viendi
+#define PRODUCT_ID      0x0877 // 8L + 1, 8L was prototype
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Viendi
+#define PRODUCT         8L
+
```

Comment:
```suggestion
```

Reply:
Committed in [2d6b639](https://github.com/qmk/qmk_firmware/pull/17943/commits/2d6b6395a87f9692f71f5688cc3b282317139697)